abempah/sample_pipelines
# Wrapped up first access of item from stock X

**### Title:**
**Feature:** Scrape Item Details from StockX Initial Test

**Description:**
This PR introduces a new feature for a stock x web scraper which now has the ability to scrape specific item details from StockX. The new features include:

1. Scraping the name of the item.
2. Scraping the last sale price of the item.

**_Here is a brief overview of the changes made:_**

Added a new spider class GrailsSpider in spiders/grails_spider.py.
- In the GrailsSpider class, defined start_urls with the StockX item URL.

- Created a new method parse_items which extracts the item name and last sale price using CSS and XPath selectors.

- This feature enhances our data collection capabilities from StockX, allowing us to gather detailed information about individual items for further analysis.

**_Test Plan:_**
To test this feature:

Ensure you are in the correct Python environment where Scrapy is installed.
From the terminal, navigate to the project directory.

- Run the spider using the following command: scrapy crawl grails

- Verify that the spider successfully prints the item name, last sale price, and URL.

Please review and let me know if there are any questions or concerns.
